modified:   src/App.css

asdf1163 opened the pull request:
modified:   src/App.js
	new file:   src/components/Header.js

asdf1163 committed:
modified:   src/App.css
	modified:   src/App.js
	new file:   src/components/Header.js

asdf1163 committed:
git commit

asdf1163 committed:
git -commit

asdf1163 committed:
git commit

asdf1163 committed:
git commit

pawelkonikpl commented on src/App.js:
```diff
@@ -2,28 +2,50 @@ import React from "react";
+    <div className="App">
+      <div className="fav"><LaterList /></div>
+      <div className="menu">
+        {Kategorie.map((option) => (<Header key={option.id} id={option.id} title={option.name} img={option.img} onZmiana={zmiana} />))}
```
Raczej korzystaj z angielskich nazw

pawelkonikpl replied:
```suggestion
        {categories.map((option) => (<Header key={option.id} id={option.id} title={option.name} img={option.img} handleChange={chane} />))}
```

pawelkonikpl replied:
Możesz tutaj użyć destrukturyzacji przykład niżej

pawelkonikpl commented on src/components/Box.js:
```diff
@@ -1,8 +1,14 @@
+import Joker from '../media/images/joker.jpg';
+//import Arrow from '../media/images/arrow.png';
+
 const Box = (props) => {
```
Możesz tutaj uzyć destrukturyzacji
```suggestion
const Box = ({ link, name }) => {
```

pawelkonikpl commented on src/components/Box.js:
```diff
@@ -1,8 +1,14 @@
-            <div>{props.name2}</div>
+        <div className='article'>
+            {/* <div className='info'><img src={Arrow} alt='brak'></img></div> */}
+            <div className="pozniej"><Later link={props.link} name_article={props.name}/></div>
```
propsy powinny by camelCase
```suggestion
            <div className="later"><Later link={link} nameArticle={name}/></div>
```

pawelkonikpl commented on src/components/Header.js:
```diff
@@ -0,0 +1,57 @@
+function Header({id, name, title, img, onZmiana})
+{
+  return (
+  <button className="option" data={id} onClick={() => onZmiana(id)}><img src={img} alt={name}></img>{title}</button>
```
english

pawelkonikpl commented on src/components/Later.js:
```diff
@@ -0,0 +1,27 @@
+      return console.log(props.link)
+    }
+    
+
```
niepotrzebna pusta linia

pawelkonikpl commented on src/components/LaterList.js:
```diff
@@ -0,0 +1,31 @@
+
+const Button = () => {
+    
+    const [b_lista, setLista] = useState(true)
```
b_lista nie za bardzo coś mówi 
Staraj sie opisać w zrozumiały sposób tak żeby kod sie sam dokumentował
i używaj cameCase

pawelkonikpl commented on src/components/LaterList.js:
```diff
@@ -0,0 +1,31 @@
+            <div className="lista"></div>
+        );
+    }
+
```
nie potrzebne puste linie i znaki

pawelkonikpl commented on src/components/LaterList.js:
```diff
@@ -0,0 +1,31 @@
+            <p onClick={() => onClick()}> {b_lista ? <BsClockHistory style={{height: '35px', width: '35px'}}/> : <BsX style={{color: 'red', height: '35px', width: '35px'}}/>}</p>
+            <>{b_lista ? '' : lista()}</>
+        </div>
+        
```
uste linei

pawelkonikpl commented on src/App.js:
```diff
@@ -2,28 +2,50 @@ import React from "react";
-    instance.get('/posts').then((response) => {
-      setPost(response.data);
-      console.log(response.data);
+  let zmiana = (id) => {
```
```suggestion
  const zmiana = (id) => {
```

pawelkonikpl commented on src/App.js:
```diff
@@ -2,28 +2,50 @@ import React from "react";
+  let zmiana = (id) => {
+    
+    return (
+      setSearch('/search?show-elements=image&api-key=86069387-0efe-47f5-a62d-bc65c5e9e7db&section='+id),
```
jeżeli musisz dodać api-key do każdego żądania spróbuj pokombinować z 
https://github.com/axios/axios#interceptors

pawelkonikpl commented on src/components/Box.js:
```diff
@@ -1,8 +1,14 @@
-            <div>{props.name}</div>
-            <div>{props.name2}</div>
+        <div className='article'>
+            {/* <div className='info'><img src={Arrow} alt='brak'></img></div> */}
```
usuń komentarze

pawelkonikpl commented on src/components/Header.js:
```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+
+function Header({id, name, title, img, onZmiana})
```
engilsh

pawelkonikpl commented on src/components/Header.js:
```diff
@@ -0,0 +1,57 @@
+{
+  return (
+  <button className="option" data={id} onClick={() => onZmiana(id)}><img src={img} alt={name}></img>{title}</button>
+  //drugi klik 
```
usuń komentarze

asdf1163 committed:
git commit

asdf1163 committed:
git commit

asdf1163 committed:
modified:   src/App.css
	modified:   src/App.js
	modified:   src/components/Box.js
	modified:   src/components/Later.js
	modified:   src/components/LaterList.js
	modified:   src/media/images/joker.jpg

asdf1163 committed:
modified:   src/App.js
	modified:   src/components/Box.js
	modified:   src/components/Later.js
	modified:   src/components/LaterList.js
